File: Java/scr/Project1.1.java
//Preston Peck
//CS 323-23/700
//September 5, 2016
//Project 1

import java.io.*;
import java.util.*;

public class Project1 {
	public static void main(String[] args) {
		try {
			Scanner inFile1 = new Scanner(new FileReader(args[0]));
			PrintWriter outFile1 = new PrintWriter(args[1]);
			linkedList CwordlistHead = new linkedList();
			String commonWord;
			while(inFile1.hasNext()) {
				commonWord = inFile1.next();
				listNode spot = CwordlistHead.findSpot(commonWord);
				listNode newNode = new listNode(commonWord);
				CwordlistHead.listInsert(spot, newNode);
				CwordlistHead.printList(outFile1);
			}
			inFile1.close();
			outFile1.close();
		}
		catch (Exception e) {
			System.out.println(e);
		}
	}
}

File: C++/Seperate Comp/Project1.1.cpp
#include "linkedList.h"
#include <iostream>
//Preston Peck
//CS 323-23/700
//September 5, 2016
//Project 1

#include <fstream>
#include <string>

int main(int argc, char *argv[]) {
	ifstream inFile1;
	ofstream outFile1;
    inFile1.open(argv[1]);
    outFile1.open(argv[2]);
    if (!inFile1.is_open()) {
        cerr<<"File failed to open!"<<endl;
        return 0;
    }
	linkedList* CwordlistHead = new linkedList();
	string commonWord;
	while(inFile1 >> commonWord) {
        if (commonWord == "I") {
		    commonWord[0] = tolower(commonWord[0]);
	    }
		listNode* spot = CwordlistHead->findSpot(commonWord);
		listNode* newNode = new listNode(commonWord);
		CwordlistHead->listInsert(spot, newNode);    
		CwordlistHead->printList(outFile1);
	}
    inFile1.close();
    outFile1.close();
}